pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -133,29 +133,12 @@ def test_asarray_tz_aware(self):
 
         tm.assert_numpy_array_equal(result, expected)
 
-    def test_CBH_deprecated(self):
-        msg = "'CBH' is deprecated and will be removed in a future version."
-
-        with tm.assert_produces_warning(FutureWarning, match=msg):
-            expected = date_range(
-                dt.datetime(2022, 12, 11), dt.datetime(2022, 12, 13), freq="CBH"
-            )
-        result = DatetimeIndex(
-            [
-                "2022-12-12 09:00:00",
-                "2022-12-12 10:00:00",
-                "2022-12-12 11:00:00",
-                "2022-12-12 12:00:00",
-                "2022-12-12 13:00:00",
-                "2022-12-12 14:00:00",
-                "2022-12-12 15:00:00",
-                "2022-12-12 16:00:00",
-            ],
-            dtype="datetime64[ns]",
-            freq="cbh",
-        )
+    @pytest.mark.parametrize("freq", ["2H", "2BH", "2S"])
+    def test_CBH_deprecated(self, freq):
```

Comment:
```suggestion
    def test_CBH_raises(self, freq):
```

Reply:
Thanks @mroeschke for reviewing this PR. I changed the name for the test.